templates/anndata_conversion.ipynb

```diff
@@ -0,0 +1,207 @@
+{
```

Comment:
<p>Does the embedded image show up correctly in the actual notebook? Not showing up here, could just be an issue with reviewnb</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/1079/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
yeah, I think it's cause the image is being displayed with html, instead of markdown's native method.

Here's what it looks like when I open the notebook in Jupyter Lab.
![image](https://github.com/angelolab/ark-analysis/assets/8909315/24975941-8481-46f8-91fa-566f41d4a600)